README.md

```diff
@@ -8,19 +8,213 @@
 
 ## Overview
 
+[HubSpot ](https://www.hubspot.com/) is an AI-powered customer platform with all the software, integrations, and resources you need to connect your marketing, sales, and customer service. HubSpot's connected platform enables you to grow your business faster by focusing on what matters most: your customers.
+
+
 [//]: # (TODO: Add overview mentioning the purpose of the module, supported REST API versions, and other high-level details.)
 
+
+The `hubspot.automation.actions` package offers APIs to connect and interact with [Hubpsot Automation action API](https://developers.hubspot.com/docs/reference/api/automation/custom-workflow-actions) endpoints.
+
 ## Setup guide
 
 [//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
```

Comment:
Remove this todo-comment (please check for other places too)

Reply:
done -https://github.com/ballerina-platform/module-ballerinax-hubspot.automation.actions/pull/2/commits/acbce5b6cc9aaec87ffb6965c63cb150edbe13ab